collections/open-source-organizations/index.md

```diff
@@ -14,6 +14,8 @@ items:
  - IBM/ibm.github.io
  - Microsoft/microsoft.github.io
  - artsy/artsy.github.io
+ - Google/google.github.io
```

Comment:
I'm not seeing the value of including [this repo](https://github.com/google/google.github.io). It just redirects to the Google organization. Help me out here.

Reply:
https://youtu.be/JrF2npRKIbQ